README.md

```diff
@@ -1,6 +1,56 @@
-# [ng-suggestion](http://ianwalter.github.io/ng-suggestion/)
+# [ng-suggestion](https://github.com/flashnotes/ng-suggestion/)
```

Comment:
Let's make this the URL of the Gh-pages demo page: `http://flashnotes.github.io/ng-suggestion/`